index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li><a href="http://">Option 1</a></li>
+                <li><a href="http://">Option 2</a></li>
+                <li><a href="http://">Option 3</a></li>
+            </ul>
+        </nav>
+    </header>
+    <div id="container">
+        <div id="sidebar">
+            <img src="./images/animals.jpg" height="50px" alt="Logo"><label>Select Video</label>
+            <ul>
+                <li><a href="http://"><img src="./images/bird.jpg" alt="Animals" width="30px"
+                            height="30px"></a><Label>   Birds</Label></li>
+            </ul>
+            <ul>
+                <li><a href="http://"></a><img src="./images/panda.jpg" alt="Animals" width="30px"
+                        height="30px"></a><Label>   Panda</Label></li>
+            </ul>
+            <ul>
+                <li><a href="http://"></a><img src="./images/squirrel.jpg" alt="Animals" width="30px"
+                        height="30px"></a><Label>   Squirrel</Label></li>
+            </ul>
```

Comment:
Please includes the attributes like width and height of the img tag in a CSS class.
It is not good practice to include these attributes within HTML